### DISCUSSION TITLE
FTX破產

### DISCUSSION THREAD

**Discussion by Aerofe** [1101-GS3073-C/Python-Community-GS3073-C#492] [General]
如果有人對此事件有興趣的話，可以看一下底下連結有相關的懶人包。大家也可以發表一下對此事件以及加密貨幣有什麼看法

懶人包：https://www.preface.ai/blog/trend/ftx-事件/

**Comment by brian431** [comment]
連第二大的交易所都會破產，由此可知不管你認為多麼地安全，雞蛋都不該放在同個籃子。

**Comment by lx3211** [comment]
這陣子常常看到有關FTX的梗圖
但平常沒接觸幣圈
現在終於了解事情完整的始末了

**Comment by manipulateed** [comment]
才剛加入區塊鏈研究社 就遇到這麼個大事件，本來想伸入投資虛擬貨幣的手 縮了回來😢

**Comment by Chenszuhong** [comment]
這感覺是一個很有趣的議題，值得學習

**Comment by xCk27x** [comment]
https://udn.com/news/story/6811/6775907
甚至連代言的知名人士都被告了...
更: 代言的部分 好像已經不起訴確定了

**Comment by xCk27x** [comment]
[FTX創辦人遭起訴多項犯罪 恐面臨最高115年刑期](https://tw.news.yahoo.com/ftx%E5%89%B5%E8%BE%A6%E4%BA%BA%E9%81%AD%E8%B5%B7%E8%A8%B4%E5%A4%9A%E9%A0%85%E7%8A%AF%E7%BD%AA-%E6%81%90%E9%9D%A2%E8%87%A8%E6%9C%80%E9%AB%98115%E5%B9%B4%E5%88%91%E6%9C%9F-091953461.html?guccounter=1&guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAAJfhpnMvWmrkZD7cquqSpAyEr8t_wWGgaAp-6gKWLKT1kNzUKssT9f6TG-OXBbZ9__qT82Phtwqes-flGc8mgc4iLufjzwrAu547-_B4D1Sz_4o8qS7SECFC_ofHVNIZpk-4ezmdHASUmuiRrzpu2j4Wv1L9W1lMC2ajndcVtis5)
更新: 看起來會被關一段時間

**Comment by williamyen0623** [comment]
這告訴我們，爆炸頭的交易所都會爆炸（沒
投資理財有風險，而加密貨幣的風險也是數一數二的高
短期內沒有要交易的資產盡量還是放在安全性較高的冷錢包比較好
雖然最近幣圈一直發生憾事，但我還是相信區塊鏈以及加密貨幣產業的未來發展，希望每個人都能好好對待這個革新的產業

**Comment by Ben6uo** [comment]
投資理財有風險 要小心憶點阿

**Comment by HsuChen88** [comment]
十萬網友都驚呆了，雖然我沒有碰虛擬貨幣，但FTX的倒下也代表著虛擬貨幣不再可靠。

**Comment by skyling0299** [comment]
投資理財都有風險，過往有金融海嘯，會不會未來也可能會出現虛擬貨幣的災難呢?

**Comment by 0212Jerry** [comment]
之前也在觀望虛擬貨幣，正想要下手的時候就發生了這件事，讓我也不得不重新思考虛擬貨幣這件事情，虛擬貨幣五花八門，投資固然以安全為主

**Comment by Vincent20030314** [comment]
如果沒有過人的膽識和財力錢，還是乖乖留著就好，不然就被當韭菜割。
